frontend/controller/actions/group.js

```diff
@@ -281,11 +301,24 @@ export default (sbp('sbp/selectors/register', {
           }
         },
         signingKeyId: CSKid,
-        encryptionKeyId: CEKid
+        encryptionKeyId: CEKid,
+        innerSigningContractID: null
```

Comment:
Does this need to be specified as `null` or can it be removed?

If it needs to be specified as `null` can you add a comment explaining why?

Reply:
Maybe it doesn't need to be `null` but it makes more sense, since the General chatroom is part of creating the group and not created by any specific user (technically, it's created by the group creator, but that's an implementation detail).